src/rules/method_length.ts

```diff
@@ -5,10 +5,16 @@ import {MethodLengthStats} from "../abap/method_length_stats";
 import {IRule} from "./_irule";
 import {BasicRuleConfig} from "./_basic_rule_config";
 
-/** Chcecks that methods do not exceed the set number of statements */
+/** Checks relating to method length. */
 export class MethodLengthConf extends BasicRuleConfig {
   /** Maximum method length in statements */
   public statements: number = 100;
+  public checkEmptyMethods: boolean = true;
```

Comment:
please add description `/** foobar */`, and suggest renaming to `errorWhenEmpty` or something similar